libraries/AP_Motors/AP_MotorsMatrix.cpp

```diff
@@ -602,6 +602,15 @@ void AP_MotorsMatrix::setup_motors(motor_frame_class frame_class, motor_frame_ty
                     add_motor(AP_MOTORS_MOT_6,-150, AP_MOTORS_MATRIX_YAW_FACTOR_CW,  4);
                     success = true;
                     break;
+                case MOTOR_FRAME_TYPE_H:
+                    // H is same as X except middle motors are closer to center
+                    add_motor_raw(AP_MOTORS_MOT_1, -0.6666f, 0.0f, AP_MOTORS_MATRIX_YAW_FACTOR_CW, 2);
```

Comment:
I think the motor position and number are the same as the HEXA X type.
I think it is not in the position of the proposal.

![motororder-hexa-x-2d](https://user-images.githubusercontent.com/646194/55931578-12e39a80-5c61-11e9-8bbb-d79f2c18aa88.png)

Reply:
@muramura, ah, yes, I've included the incorrect diagram in the picture above.  I've corrected the diagram to match the existing HexaX frame.